addons/rhs/CfgMagazines.hpp

```diff
@@ -459,7 +459,6 @@ class CfgMagazines
         mass = 77.14; // 60
     };
 
-
     class ClaymoreDirectionalMine_Remote_Mag : CA_Magazine // Claymore
```

Comment:
Warum ist das eigl im RHS Mod?
Dann ist es nicht drin, wenn wir mal ohne RHS spielen... das sind doch alles vanilla Klassen oder?

Reply:
Du meinst die Wiesel 2? Die habe ich wie alle anderen Fahrzeuge in /rhs/CfgVehicles gepackt, weil da bereits alle anderen Vanilla Modelle enthalten sind. Wir können das aber auch gerne noch ändern. Willst du alles was Vanilla angeht unter /skins und /main gepackt haben?

Oder geht es um die Claymore? Falls ja, wohin soll ich sie packen? Unter /main/cfgVehicles?

Reply:
Für redskins wäre es skins wenn es kein RHS Fahrzeug ist wie bereits die Sek fahrzeuge

Reply:
Nur skins und Skin Anpassungen bitte in Skins
ALLES andere was Vanilla Configs sind in main... oder ein extra addon namens config wo wir alles Configspezifische rein machen
@Er1807 main mit configs zu ballern oder extra addon namens config mit cfgVehicle/cfgmag usw?!?

BTW: mir gings um beides, in RHS nur configs und sachen die aus RHS kommen und nur damit gehen, sry für das missverständnis hierbei und generell